import-spreadsheets.qmd

```diff
@@ -1,11 +1,26 @@
-# Spreadsheets {#import-spreadsheets}
+# Spreadsheets {#sec-import-spreadsheets}
+
+```{r}
+#| label: setup
```

Comment:
I think the `setup` label should be applied to the code in the prereqs, because that's what you actually want run when editing interactively.

Reply:
Relabeled these chunks to `source-common` throughout. When https://github.com/yihui/knitr/issues/2126 is fixed it knitr wouldn't allow two chunks with `setup` label anyway.